plugin/storage/es/options.go

```diff
@@ -217,17 +217,17 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 		"Optional date separator of Jaeger indices. For example \".\" creates \"jaeger-span-2020.11.20\".")
 	flagSet.String(
 		nsConfig.namespace+suffixIndexRolloverFrequencySpans,
-		defaultIndexRolloverFrequency,
+		nsConfig.Indices.Spans.RolloverFrequency,
 		"Rotates jaeger-span indices over the given period. For example \"day\" creates \"jaeger-span-yyyy-MM-dd\" every day after UTC 12AM. Valid options: [hour, day]. "+
 			"This does not delete old indices. For details on complete index management solutions supported by Jaeger, refer to: https://www.jaegertracing.io/docs/deployment/#elasticsearch-rollover")
 	flagSet.String(
 		nsConfig.namespace+suffixIndexRolloverFrequencyServices,
-		defaultIndexRolloverFrequency,
+		nsConfig.Indices.Services.RolloverFrequency,
 		"Rotates jaeger-service indices over the given period. For example \"day\" creates \"jaeger-service-yyyy-MM-dd\" every day after UTC 12AM. Valid options: [hour, day]. "+
 			"This does not delete old indices. For details on complete index management solutions supported by Jaeger, refer to: https://www.jaegertracing.io/docs/deployment/#elasticsearch-rollover")
 	flagSet.String(
 		nsConfig.namespace+suffixIndexRolloverFrequencySampling,
-		defaultIndexRolloverFrequency,
+		nsConfig.Indices.Sampling.RolloverFrequency,
```

Comment:
@yurishkuro these were being set to default here previously so I changed them to use the set values - is this fine to do or do we want to leave these as default

Reply:
if the respective fields in the struct are initialized to that default then it's fine. You can validate by running `binary help`, which outputs the default values - if they are the same before/after the change then you're good.

Reply:
Got it! I ran `SPAN_STORAGE_TYPE=elasticsearch go run ./cmd/collector help` on main and on this branch and the rollover frequency is defaulted to `day` on both.